project/LibDependencies.scala

```diff
@@ -3,7 +3,7 @@ import sbt.*
 
 object LibDependencies {
 
-  val playFrontendHmrcVersion = "9.6.0"
+  val playFrontendHmrcVersion = "10.3.0"
```

Comment:
Is this not meant to be 10.4.0?

Reply:
@kyle-bowden I have just sworn very loudly. Thanks, converted back to draft.